blocks/model.py

```diff
@@ -190,7 +179,13 @@ def get_params(self):
         parameters that do not belong to any brick.
 
         """
-        return self.params
+        if (len(set(parameter.name for parameter in self.parameters)) !=
+                len(self.parameters)):
+            raise ValueError("parameters should have different names")
```

Comment:
I see. So should I extract parameters' hierarchical names and then check whether they are unique or not?
If so, should I loop over bricks in ```self.top_bricks``` and find names of parameters?